lab1/print_style.css

```diff
@@ -0,0 +1,83 @@
+body {
+  padding: 50px 100px;
+  background-color: white;
+  font-family: "Comic Sans MS", arial;
+  font-size: 100%;
```

Comment:
Alright, but I think then you should put it on a fixed value, as it wont change anything right now. Just put it on 14px or something.